tensorflow/core/util/cuda_kernel_helper.h

```diff
@@ -47,16 +129,22 @@ struct CudaLaunchConfig {
 // memory-limited.
 inline CudaLaunchConfig GetCudaLaunchConfig(int work_element_count,
                                             const GPUDevice& d) {
+  CudaLaunchConfig config;
+
+  // in case of invalid input, return the default value config, which has all -1
+  if (work_element_count <= 0) {
+    return config;
+  }
```

Comment:
Which is better here? Log and exit using `CHECK_GE`, or explicitly return an invalid output as in above?